trademapper/js/trademapper.csv.js

```diff
@@ -74,7 +74,7 @@ define([
 
 	loadCSVFile: function() {
 		this.resetLoadErrors();
-		this.csvFile = this.fileInputElement[0][0].files[0];
+		this.csvFile = this.fileInputElement._groups[0][0].files[0];
```

Comment:
Is this accessing a private member? Is there no other way?

Reply:
Might be possibly to get at it, yes. The issue is that this is a d3 select, so the API is a bit odd and I couldn't find the docs. I'll have another look at it and see if I missed something.

Reply:
Fixed, used d3.event.target.files instead, which is the right way to do it.